manage_breast_screening/mammograms/forms/appointment_note_form.py

```diff
@@ -27,3 +27,7 @@ def save(self):
         self.instance.content = self.cleaned_data["content"]
         self.instance.save()
         return self.instance
+
+    @property
+    def instance_is_saved(self):
+        return self.instance and not self.instance._state.adding
```

Comment:
could this be simplified to `self.instance.pk is not None`? Just would rather avoid depending on internals if possible

Reply:
Started off as that but it looks like Django assigns uuid primary keys at instantiation time so this check wasn't working as intended.